plugins/source-kafka/README.md

```diff
@@ -53,26 +53,26 @@ Manage your Knative Kafka eventing sources
 
 #### SEE ALSO
 
-* [kafka create](#kafka-create)	 - create NAME
-* [kafka delete](#kafka-delete)	 - delete NAME
-* [kafka describe](#kafka-describe)	 - describe NAME
+* [kafka create](#kafka-create)	 - Create a kafka source
+* [kafka delete](#kafka-delete)	 - Delete a Kafka source
+* [kafka describe](#kafka-describe)	 - Describe a Kafka source
```

Comment:
inconsistent spell for "kafka", should begin with lower case

Reply:
fixed.